client/a8c-for-agencies/sections/signup/signup-v2/components/multi-step-form/contact-form/hooks/use-contact-form-validation.ts

```diff
@@ -0,0 +1,81 @@
+import emailValidator from 'email-validator';
+import { useTranslate } from 'i18n-calypso';
+import { useCallback, useState } from 'react';
+import { isSiteActive } from 'calypso/a8c-for-agencies/components/form/utils';
+import { AgencyDetailsSignupPayload } from 'calypso/a8c-for-agencies/sections/signup/types';
+import { preventWidows } from 'calypso/lib/formatting';
+
+export const CAPTURE_URL_RGX =
+	/^(http:\/\/www\.|https:\/\/www\.|http:\/\/|https:\/\/)?[a-z0-9]+([-.][a-z0-9]+)*\.[a-z]{2,63}(:[0-9]{1,5})?(\/.*)?$/i;
+const CAPTURE_SOCIAL_URL_RGX =
+	/^(https?:\/\/)?(www\.)?(facebook\.com|linkedin\.com|instagram\.com)(\/.*)?$/i;
```

Comment:
I don't know whether that's needed (won't the backend handle this?), but in any case we should try to avoid duplication here.

I see this regex is available here:

```
import { CAPTURE_URL_RGX } from 'calypso/blocks/import/util';
```